pd_django_small/organizations/models.py

```diff
@@ -0,0 +1,17 @@
+import uuid
+from django.db import models
+from shortuuidfield import ShortUUIDField
+
+
+class Organization(models.Model):
+    uuid = models.UUIDField(
+        unique=True,
+        primary_key=True,
+        default=uuid.uuid4,
+        editable=False,
+    )
+    name = models.CharField(max_length=255, blank=False)
+    owner = ShortUUIDField(
```

Comment:
it is better to use one type of UUID everywhere, I would stop on **models.UUIDField** and kill **ShortUUIDField**